### DISCUSSION TITLE
Login problem

### DISCUSSION THREAD

**Discussion by Belardino** [CA17/TeamsACS#16] [General]
hi, during login, it does not accept the default username and password 'admin/teamsacs' . Returns 'User does not exist'.

**Comment by jamiesun** [comment]
It is possible that the database is not successfully connected

**Reply by yljiang28** [reply]
The same problem occurs. How do I check if the database is successfully connected?

**Comment by yljiang28** [comment]
After completing the installation following the quick installation instructions, ps aux did not find the teamsacs process。
